vassal-app/src/main/java/VASSAL/tools/io/ZipArchive.java

```diff
@@ -425,18 +427,50 @@ private void moveFile(Path src, Path dst) throws IOException {
         // attempt to copy to the destination
         Files.copy(src, dst, StandardCopyOption.REPLACE_EXISTING);
       }
-      catch (IOException e) {
-        // copy failed
-        final String fmt = "Unable to overwrite %s, so data written to %s instead: %s";
-        throw new IOException(
-          String.format(
-            fmt,
-            dst.toAbsolutePath(),
-            src.toAbsolutePath(),
-            e.getMessage()
-          ),
-          e
-        );
+      catch (IOException copyException) {
+        Path dst2 = dst.resolveSibling(src.getFileName()); // Our "temp" name but in the destination directory
+        final String fmt = Resources.getString("Editor.ZipArchive.overwrite") + "\n%s"; //NON-NLS
```

Comment:
Can be moved down and inward, nearer to where it's needed?

Reply:
I just tried, and then remembered how it ended up here -- it is used for BOTH of the following "catch" statements, so has to be above the first try?

Reply:
Oh, I see. I misread the nesting.